patches/extensions-browser-extension_event_histogram_value.h.patch

```diff
@@ -20,6 +20,7 @@ index d29bd5d6776c488a5d24dc40ca102946044131f9..b953ecdf34dd83ffce42347785b38c38
 +  BRAVE_REWARDS_GET_NOTIFICATION,
 +  BRAVE_REWARDS_GET_ALL_NOTIFICATIONS,
 +  BRAVE_WALLET_FAILED,
++  BRAVE_WALLET_CORRUPTED,
```

Comment:
patch is not needed as we use `BRAVE_START` for all of them

Reply:
Removed and code adjusted.